File: to-do-app-ui/src/app/models/todo.item.ts
export class ToDoItem {
    public id: number;
    public toDoTask: string;
    public isCompleted: boolean;
    public actualisationDate: string;

    constructor(id: number, toDoTask: string, isCompleted: boolean, actualisationDate: string) {
        this.id = id;
        this.toDoTask = toDoTask;
        this.isCompleted = isCompleted;
        this.actualisationDate = actualisationDate;
    }
}

File: ToDoDAL/Models/ToDoItem.cs
using System;
using System.Collections.Generic;
using System.Text;
using ToDoDAL.Interfaces;


namespace ToDoListApp_DAL.Models
{
    public class ToDoItem: IToDoItem
    {
        public int Id { get; set; }
        public string ToDoTask { get; set; } 
        public bool IsCompleted { get; set; }
        public DateTime ActualisationDate { get; set; }
    }
}
